### ISSUE TITLE
Using node-sqlite3 with Node 0.11.9

### ISSUE THREAD

**Issue by toptierlabs** [mapbox/node-sqlite3#524]
I know that there are some incompatibilities with this version but I am not able to update node on this production app.

Is there any way that I can make them play nicely.
It has to work with node-webkit v0.9.2 and node 0.11.9.

This is the error that I get:

I am executing
**npm install sqlite3 --build-from-source --runtime=node-webkit --target_arch=$1 --target=0.9.2 --update-binary**

And getting the following error

In file included from ../node_modules/nan/nan_new.h:189:
../node_modules/nan/nan_implementation_12_inl.h:150:59: error: expected '(' for function-style cast or type construction
                              , value).As<v8::NumberObject>();
                                          ~~~~~~~~~~~~~~~~^
../node_modules/nan/nan_implementation_12_inl.h:150:61: error: expected expression
                              , value).As<v8::NumberObject>();

Bottom line is that my versions of node and node-webkit are frozen.

Is there any branch that I can look at?

**Comment by springmeyer** [comment]
The first 0.11.x version we attempted to support was 0.11.13 (https://github.com/mapbox/node-sqlite3/blob/master/CHANGELOG.md#224). Because even patch versions of 0.11.x broke compiles it was very difficult to impossible to consistently support other 0.11.x versions. So, obviously being frozen not only to 0.11.x but an old 0.11.x series is going to be challenging for you. I recommend you use 0.12 and if you can't then find some alternative for persistance that is not this module.
